src/components/Nav/Nav.jsx

```diff
@@ -1,10 +1,29 @@
+import { useEffect, useState } from 'react';
 import {Link} from 'react-router-dom';
+import requests from '../../utils/requests';
+import './Nav.css';
 
 const Nav = () => {
-    return <div>
-        <Link to='/'>Home</Link>
-        <Link to='/articles'>All articles</Link>
-    </div>
+    const [topics, setTopics] = useState();
+    const [isLoading, setIsLoading] = useState(true);
+    useEffect(() => {
+        setIsLoading(true);
+        requests.getTopics()
+        .then((res) => {
+            setTopics(res);
+            setIsLoading(false);
+        })
+    },[])
+
+    return <div className='navBar'>
+        <Link className='navBarLink' to='/'>Home</Link>
+        <Link className='navBarLink' to='/articles'>All articles</Link>
+        {isLoading?<></>:
```

Comment:
if you don't want to display something use null rather than rendering a fragment